### PR TITLE
Add UST and LUNC to Fantom & Polygon

### PR DESCRIPTION
**Pull request by fahimahmedx**
Brief comments on the purpose of your changes:
Adding UST and LUNC to Fantom chain & Polygon chain

**For Dune Engine V2**

I've checked that:
I've checked that the tokens are on coinpaprika and that the contract addresses are correct & in lowercase.

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/prices/prices_tokens.sql [hunk 1]
```diff
@@ -1774,6 +1774,8 @@ VALUES
     ("nuls-nuls", "polygon", "NULS", "0x8b8e48a8cc52389cd16a162e5d8bd514fabf4ba0", 8),
     ("blank-blockwallet","polygon", "BLANK", "0xf4C83080E80AE530d6f8180572cBbf1Ac9D5d435", 18),
     ("fin-fin", "polygon", "FIN", "0x576c990a8a3e7217122e9973b2230a3be9678e94", 18),
+    ("luna-luna-wormhole", "polygon", "LUNA", "0x9cd6746665d9557e1b9a775819625711d0693439", 6),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:1777]: https://polygonscan.com/token/0x9cd6746665d9557e1b9a775819625711d0693439

File: models/prices/prices_tokens.sql [hunk 2]
```diff
@@ -1774,6 +1774,8 @@ VALUES
...
+    ("ust-terrausd", "polygon", "USTC", "0x692597b009d13c4049a947cab2239b7d6517875f", 18),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:1778]: https://polygonscan.com/token/0x692597b009d13c4049a947cab2239b7d6517875f

File: models/prices/prices_tokens.sql [hunk 3]
```diff
@@ -1793,7 +1795,7 @@ VALUES
     ("steth-lido-staked-ether", "solana", "wstETH", "ZScHuTtqZukUrtZS43teTKGs2VqkKL8k4QCouR2n6Uo", 8),
     ("stsol-lido-staked-sol", "solana", "stSOL", "7dHbWXmci3dT8UFYWYZweBLXgycu7Y3iL6trKn1Y7ARj", 9),
     ("sol-solana", "solana", "SOL", "So11111111111111111111111111111111111111112", 9),
-    ("ust-terrausd", "solana", "UST", "9vMJfxuKxXBoEa7rM12mYLMwTacLMLDJqHozw96WQL8i", 6),
+    ("ust-terrausd", "solana", "USTC", "9vMJfxuKxXBoEa7rM12mYLMwTacLMLDJqHozw96WQL8i", 6),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:1798]: https://explorer.solana.com/address/9vMJfxuKxXBoEa7rM12mYLMwTacLMLDJqHozw96WQL8i

File: models/prices/prices_tokens.sql [hunk 4]
```diff
@@ -1849,7 +1851,9 @@ VALUES
     ("3omb-3omb-token", "fantom", "3OMB", "0x14DEf7584A6c52f470Ca4F4b9671056b22f4FfDE", 18), 
     ("gscarab-gscarab", "fantom", "GSCARAB", "0x6ab5660f0B1f174CFA84e9977c15645e4848F5D6", 18), 
     ("lif3-lif3", "fantom", "LIF3", "0xbf60e7414EF09026733c1E7de72E7393888C64DA", 18),
-    ("oxd-0xdao", "fantom", "OXD", "0xc165d941481e68696f43EE6E99BFB2B23E0E3114", 18), 
+    ("oxd-0xdao", "fantom", "OXD", "0xc165d941481e68696f43EE6E99BFB2B23E0E3114", 18),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:1854]: https://ftmscan.com/token/0xc165d941481e68696f43EE6E99BFB2B23E0E3114

File: models/prices/prices_tokens.sql [hunk 5]
```diff
@@ -1849,7 +1851,9 @@ VALUES
...
+    ("ust-terrausd", "fantom", "USTC", "0xe2d27f06f63d98b8e11b38b5b08a75d0c8dd62b9", 6),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:1855]: https://ftmscan.com/token/0xe2d27f06f63d98b8e11b38b5b08a75d0c8dd62b9

File: models/prices/prices_tokens.sql [hunk 6]
```diff
@@ -1849,7 +1851,9 @@ VALUES
...
+    ("luna-terra", "fantom", "LUNC", "0x95dd59343a893637be1c3228060ee6afbf6f0730", 6),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:1856]: https://ftmscan.com/token/0x95dd59343a893637be1c3228060ee6afbf6f0730

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @fahimahmedx 

I comment the explorer links for tokens below.

**Review by soispoke — APPROVED** [review]

**Pull request merged**